### ISSUE TITLE
Lens unable to connect to cluster

### ISSUE THREAD

**Issue by KenBurk** [lensapp/lens#6432]
**Describe the bug**
One of 5 clusters in Lens will not connect. Lens will sping with "Connecting..." instead of loading cluster
tried deleting lens-local-storage folder

**To Reproduce**
1. Go to Clusters
2. Click on cluster

**Expected behavior**
Selecting cluster will load

**Screenshots**
![image](https://user-images.githubusercontent.com/91329150/196766710-3e04248d-9080-4c57-9180-fade9b8d7029.png)

**Environment (please complete the following information):**
- Lens Version: 2022.10.131529-latest
- OS: Windows 10
- Installation method (e.g. snap or AppImage in Linux):
MSI

**Comment by Nokel81** [comment]
I assume no other text is displayed? Would you be able to provide logs from `%AppData%\Lens`?

**Comment by KenBurk** [comment]
[lens15.log](https://github.com/lensapp/lens/files/9823157/lens15.log)

**Comment by Nokel81** [comment]
Which cluster doesn't display anything?

**Comment by KenBurk** [comment]
aks-blue-cluster

**Comment by KenBurk** [comment]
kubectl has no problem against this server

**Comment by Nokel81** [comment]
Looking at your logs I see some lines like:

```
error: [CLUSTER]: Failed to connect to "aks-blue-cluster": StatusCodeError: 500 - "dial tcp 20.96.130.143:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.\n"
```

If you quit Lens from tray icon and start it via a terminal does connecting work?

**Comment by KenBurk** [comment]
Yes, that is working. Thanks!

**Comment by Nokel81** [comment]
Do you do any set up in your terminal before starting lens?

**Comment by KenBurk** [comment]
No

**Comment by Nokel81** [comment]
Interesting, windows processes inherent all environmental variables from its parent process. So I guess the only way would be if you modified the environment somewhere after starting Lens I guess.

**Comment by KenBurk** [comment]
FWIW, a colleague said he was having the problem and I gave him the same instructions and it fixed his issue as well.

**Comment by Nokel81** [comment]
That is very strange... Are you using PowerShell?

**Comment by Nokel81** [comment]
Do you have a PowerShell profile set up?

**Comment by KenBurk** [comment]
No, I usually drop to WSL and use bash shell for local terminal

**Comment by Nokel81** [comment]
So you started Lens from within `WSL` or did you run it from PowerShell? Given how windows processes inherit environment I am at a loss as to where the additional environment variables are coming from allowing you to connect when running Lens via the terminal.

**Comment by KenBurk** [comment]
Run directly from start menu

**Comment by Nokel81** [comment]
Does running Lens from the start menu after quitting it from the tray menu also solve the problem?

**Comment by KenBurk** [comment]
yes, quitting the service from the tray is the workaround

**Comment by Nokel81** [comment]
Okay so it sounds like that is what is needed to pick up environment variable changes. Thanks for the response.
